demo/kind-cluster-localhost.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+  - extraPortMappings:
+      - containerPort: 30443
+        hostPort: 30443
+        protocol: TCP
+    image: kindest/node:v1.18.8@sha256:f4bcc97a0ad6e7abaf3f643d890add7efe6ee4ab90baeb374b4f41a4c95567eb
```

Comment:
Do we want to pin the Kubernetes version? Anyone going through the reana.io instructions will run kind and therefore Kubernetes in the latest possible version. In case we want to pin we should pin to `1.19` once `0.7.0-alpha.2` is out.

Reply:
Would be better not to pin, so far REANA installs with a variety of versions... Also I'm using different kind version locally, would be fun to see how the above `image` will cooperate...